libs/langchain/langchain/chains/combine_documents/base.py

```diff
@@ -15,19 +16,62 @@
 from langchain.chains.base import Chain
 from langchain.text_splitter import RecursiveCharacterTextSplitter, TextSplitter
 
-DEFAULT_DOCUMENT_SEPARATOR = "\n\n"
+""" --- Constants --- """
 DOCUMENTS_KEY = "context"
+INTERMEDIATE_STEPS_KEY = "intermediate_steps"
+
+""" --- Defaults --- """
 DEFAULT_DOCUMENT_PROMPT = PromptTemplate.from_template("{page_content}")
+DEFAULT_DOCUMENT_SEPARATOR = "\n\n"
+
 
+""" --- Utilities (private) --- """
```

Comment:
if private, then add `_` to start?

Reply:
it's imported in other files just not exposed in combine_documents.init. should still prefix with `_` in that case?